blueoil/networks/classification/quantize_example.py

```diff
@@ -0,0 +1,147 @@
+from functools import partial
+
+import tensorflow as tf
+
+from blueoil.blocks import darknet
+from blueoil.networks.classification.base import Base
+from blueoil.layers import conv2d
+
+
+class FooNetwork(Base):
+    """Example model with simple layer"""
+
+    def __init__(
+            self,
+            *args,
+            **kwargs
+    ):
+        super().__init__(
+            *args,
+            **kwargs,
+        )
+
+        self.activation = lambda x: tf.nn.leaky_relu(x, alpha=0.1, name="leaky_relu")
+        self.before_last_activation = self.activation
+
+    def base(self, images, is_training):
```

Comment:
In example, we can assume just one data format.

Reply:
NHWC is choosed.

Reply:
nice :)